src/pages/library/DataBrowserPreview.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import { Select } from 'src/components/common'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, spinner } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+import { SimpleTable } from 'src/components/table'
+import { Ajax } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { withErrorReporting } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+import { useDataCatalog } from 'src/pages/library/dataBrowser-utils'
+
+
+const styles = {
+  table: {
+    header: {
+      color: colors.accent(),
+      height: '1rem',
+      textTransform: 'uppercase', fontWeight: 600, fontSize: '0.75rem'
+    }
+  }
+}
+
+const activeTab = 'browse & explore'
+let metadata
+let tableMap
+
+const selectTable = ({ id, value, signal, setSelectedTable, setPreviewData, loading, setLoading }) => {
+  const loadTable = _.flow(
+    Utils.withBusyState(setLoading),
+    withErrorReporting('Error loading table')
+  )(async () => {
+    setSelectedTable(value)
+    const previewTableData = await Ajax(signal).DataRepo.getPreviewTable({
+      id, limit: 50, offset: 0,
+      table: value
+    })
+    setPreviewData(previewTableData?.result || [])
+  })
+  loadTable()
+}
+
+const DataBrowserPreview = ({ id }) => {
+  const signal = Utils.useCancellation()
+  const [loading, setLoading] = useState()
+  const { dataCatalog } = useDataCatalog()
+  const dataMap = _.keyBy('dct:identifier', dataCatalog)
+  const snapshot = dataMap[id]
+  const [tables, setTables] = useState()
+  const [selectedTable, setSelectedTable] = useState('')
+  const [previewData, setPreviewData] = useState()
+
+  Utils.useOnMount(() => {
+    const loadData = async () => {
+      metadata = await Ajax(signal).DataRepo.getPreviewMetadata(id)
+      tableMap = _.keyBy('name', metadata.tables)
+      setTables(_.map('name', metadata.tables))
+      selectTable({ id, value: metadata.tables[0].name, setSelectedTable, setPreviewData, loading, setLoading, signal })
+    }
+    loadData()
+  })
+
+  return h(FooterWrapper, { alwaysShow: true }, [
+    libraryTopMatter(activeTab),
+    !snapshot ?
+      centeredSpinner() :
+      div({ style: { padding: 20 } }, [
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', width: '100%', lineHeight: '26px' } }, [
```

Comment:
Implicit pixels here?
```suggestion
        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', width: '100%', lineHeight: 26 } }, [
```

Reply:
Actually, i made this change, but I need to change it back. For whatever reason, the line-height field does not add the pixels to the value, and the view is dramatically different by excluding this